src/main/java/edu/illinois/library/cantaloupe/processor/PdfBoxProcessor.java

```diff
@@ -191,17 +191,15 @@ private MemoryUsageSetting getMemoryUsageSetting() {
         final boolean scratchFileEnabled =
                 config.getBoolean(Key.PROCESSOR_PDF_SCRATCH_FILE_ENABLED, false);
         final long maxMainMemoryBytes =
-                config.getLong(Key.PROCESSOR_PDF_MAX_MEMORY_BYTES, -1);
-        final long maxStorageBytes =
-                config.getLong(Key.PROCESSOR_PDF_MAX_STORAGE_BYTES, -1);
+                config.getLongBytes(Key.PROCESSOR_PDF_MAX_MEMORY_BYTES, -1);
         final String scratchFileLocation =
-                config.getString(Key.PROCESSOR_PDF_SCRATCH_LOCATION);
+                config.getString(Key.TEMP_PATHNAME);
```

Comment:
Sorry, this is my fault for not thinking about it, but the way to get the canonical temp path is via `Application.getTempPath()` which will fall back to `java.io.tmpdir` if  `temp_path` is not set.

Reply:
Nice clean way of handling that!  I'll update this now.